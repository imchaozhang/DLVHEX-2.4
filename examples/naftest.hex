notactive.
nonactive.
active.
% if 'notactive' is parsed as 'not active', we won't get notworks
notworks :- notactive.
% if 'nonactive' is parsed as 'not active', we won't get nonworks
nonworks :- nonactive.
