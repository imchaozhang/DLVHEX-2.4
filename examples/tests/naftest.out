{nonworks, notworks, active, nonactive, notactive}
